library/ssl_misc.h

```diff
@@ -1507,7 +1507,7 @@ int mbedtls_ssl_psk_derive_premaster(mbedtls_ssl_context *ssl,
 #endif /* MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED */
 
```

Comment:
Please add [changelog entries](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog.d/00README.md) for the bug fixes.

Reply:
Done in 4d0b847. Please let me know in case you'd like some changes there.